integration_tests/steps/integration_tests.py

```diff
@@ -821,3 +840,17 @@ def assert_trade_rates(context, market_name, trade_rate):
     for market in markets:
         for t in market.trades:
             assert isclose(t.offer.price / t.offer.energy, float(trade_rate))
+
+
+@then('the unmatched loads are identical no matter if the past markets are kept')
+def identical_unmatched_loads(context):
+    unmatched_loads = context.simulation.endpoint_buffer.unmatched_loads
+    unmatched_loads_redis = context.simulation.endpoint_buffer.unmatched_loads_redis
+
+    assert len(DeepDiff(unmatched_loads, context.unmatched_loads)) == 0
+    # The 2 simulation runs do not assign the same uuids to the same areas
+    # therefore we have to search whether there are elements with the same values
+
```

Comment:
Please move the blank line to the top of this comment (was confused where to connect this comment to) Thanks!

Reply:
Will do, thanks!